generator/metrics_utils.py

```diff
@@ -13,7 +13,7 @@ class _MetricsConfigLoader:
     """Loads metric config files from an external repository."""
 
     config_collection: Optional[ConfigCollection] = None
-    repos: List[str] = [METRIC_HUB_REPO]
+    repos: List[str] = [METRIC_HUB_REPO, LOOKER_METRIC_HUB_REPO]
```

Comment:
This change seems unrelated to the parallelization implementation.  Is this a bug fix, or some other intentional change in LookML generation behavior?

Reply:
Not fixing a bug, but those should be the default repos to load metrics from